src/test/java/com/jcabi/github/mock/MkGithubTest.java

```diff
@@ -179,4 +180,19 @@ public Void call() throws Exception {
             Matchers.<Issue>iterableWithSize(threads)
         );
     }
+
+    /**
+     * Can retrieve users.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void canRetrieveUsers() throws Exception {
+        MatcherAssert.assertThat(
+            "Retrieved inexistent user",
+            new User.Smart(
+                new MkGithub().users().get("other")
+            ).exists(),
+            new IsEqual<>(true)
```

Comment:
@paulodamaso but, according to the ticket's [description](https://github.com/jcabi/jcabi-github/issues/1370#issue-306858750) this should be ``false`` since the user doesn't actually exist

Reply:
@amihaiemil Sorry, I didn't noticed that. The problems: `MkUsers().get("")` is creating a new user even when we does not have this user in "database". I've added a method `add(login)`  to create the user and removed creating part of `get(login)`.